# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/style/test/test_import_preload.html
# Commit: 9a8abd87cc79
# Full Hash: 9a8abd87cc7935f29b94248c1a6f8203faa14403
# Author: Henri Sivonen <hsivonen@hsivonen.fi>
# Date: 2021-12-08 21:43:06
# Regressor Bug: 1701828
# File Overlap Count: 10
# Description:
#   Bug 1701828 - meta charset rewrite. r=smaug
#   
#   Implements https://github.com/whatwg/html/issues/6962 . Improves performance
#   when <meta charset> occurs in head but after the first kilobyte and aligns
#   behavior better with WebKit and Blink.
# ==============================================================================

diff --git a/layout/style/test/test_import_preload.html b/layout/style/test/test_import_preload.html
--- a/layout/style/test/test_import_preload.html
+++ b/layout/style/test/test_import_preload.html
@@ -1,9 +1,10 @@
 <!doctype html>
+<meta charset="utf-8">
 <script src="/tests/SimpleTest/SimpleTest.js"></script>
 <link rel="stylesheet" href="/tests/SimpleTest/test.css"/>
 <script src="slow_load.sjs"></script>
 <script>
   let time = Date.now();
   SimpleTest.waitForExplicitFinish();
 
   onload = function() {